### PR TITLE
chore(docs): add full stack course (Next.js/NextAuth.js/Supabase/Prisma)

### PR DESCRIPTION
**Pull request by gdangelo**
## Reasoning 💡

Adding a new resource to the documentation website. It is a free online course created by me and teaches about building a full-stack app with Next.js. Of course, the authentication part is implemented using NextAuth.js 😉

## Checklist 🧢

- [x ] Documentation
- ~[ ] Tests~
- [ x] Ready to be merged

### CODE REVIEW

**Review by balazsorban44 — DISMISSED** [review]
Thanks! I'll review the course, but first, could we tone down the item in the list a bit? :sweat_smile: 

image


https://next-auth-git-fork-gdangelo-patch-1-nextauthjs.vercel.app/tutorials

File: docs/docs/tutorials.md [hunk 1]
```diff
@@ -51,6 +51,22 @@ title: Tutorials and Explainers
 
 ## Fullstack
 
+### [[FREE COURSE] Build a FullStack App with Next.js, Supabase & Prisma](https://themodern.dev/courses/build-a-fullstack-app-with-nextjs-supabase-and-prisma-322389284337222224)
```
Comment (balazsorban44) [inline docs/docs/tutorials.md:54]:
```suggestion
### Build a FullStack App with Next.js, NextAuth.js, Supabase & Prisma](https://themodern.dev/courses/build-a-fullstack-app-with-nextjs-supabase-and-prisma-322389284337222224)
```

**Comment by balazsorban44** [comment]
Could you give me access to the course, or at least the source code if available somewhere? :pray:

**Comment by gdangelo** [comment]
> Could you give me access to the course, or at least the source code if available somewhere? 🙏

Hey Balázs, 

Thanks for your feedback. You can access the course from [here](https://themodern.dev/courses/build-a-fullstack-app-with-nextjs-supabase-and-prisma-322389284337222224) and just click **Enroll now**. It's totally free. You have to log in first, though.

**Comment by balazsorban44** [comment]
:facepalm::sweat_smile: It did not occur to me that I could just enroll since it's free... Thanks, I'll check it out. :+1:

**Comment by ndom91** [comment]
@gdangelo we'd be happy to include the course in our list, but can you please adjust it to match the formatting of all other resources in the page? I.e. no giant banner image or `[FREE COURSE]` prefix? Thanks!

**Comment by gdangelo** [comment]
> @gdangelo we'd be happy to include the course in our list, but can you please adjust it to match the formatting of all other resources in the page? I.e. no giant banner image or `[FREE COURSE]` prefix? Thanks!

Thanks for your feedback. I've adjusted the formatting of this new entry to comply with all the other resources.

Please let me know if there is anything else I should do.

**Pull request merged**